*** html.py.copy	2006-05-16 18:36:21.000000000 -0700
--- html.py	2006-05-16 18:39:33.000000000 -0700
***************
*** 2174,2184 ****
          else: private.write(anchor)
          if puid.is_public():
              if link_public: public.write(link)
              else: public.write(anchor)
          str = psum
!         if (self._inheritance != 'grouped' and inherit):
              cls = puid.cls()
              str += ('    <i>(Inherited from %s)</i>\n' %
                      self._uid_to_href(cls, cls.shortname()))
          str += '</td></tr>'
          private.write(str)
--- 2174,2184 ----
          else: private.write(anchor)
          if puid.is_public():
              if link_public: public.write(link)
              else: public.write(anchor)
          str = psum
!         if (self._inheritance != 'grouped' and inherited):
              cls = puid.cls()
              str += ('    <i>(Inherited from %s)</i>\n' %
                      self._uid_to_href(cls, cls.shortname()))
          str += '</td></tr>'
          private.write(str)
